slack/web/classes/messages.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import List
+
+from .attachments import Attachment
+from .blocks import Block
+from .objects import JsonObject, JsonValidator, extract_json
+
+
+class Message(JsonObject):
+    attachments_max_length = 100
+
+    def __init__(
+        self,
+        *,
+        text: str,
+        attachments: List[Attachment] = None,
+        blocks: List[Block] = None,
+        markdown: bool = True,
+    ):
+        """
+        Create a message.
+
+        https://api.slack.com/messaging/composing#message-structure
+
+        Args:
+            text: Plain or Slack Markdown-like text to display in the message.
+            attachments: A list of Attachment objects to display after the rest of
+                the message's content. More than 20 is not recommended, but the actual
+                limit is 100
+            blocks: A list of Block objects to attach to this message. If
+                specified, the 'text' property is ignored (more specifically, it's used
+                as a fallback on clients that can't render blocks)
+            markdown: Whether to parse markdown into formatting such as
+                bold/italics, or leave text completely unmodified.
+        """
+        self.text = text
+        self.attachments = attachments or []
+        self.blocks = blocks or []
+        self.markdown = markdown
+
+    @JsonValidator(
+        f"attachments attribute cannot exceed {attachments_max_length} items"
+    )
+    def attachments_length(self):
+        return (
+            self.attachments is None
+            or len(self.attachments) <= self.attachments_max_length
+        )
+
+    def get_json(self) -> dict:
+        self.validate_json()
+        if len(self.text) > 40000:
+            logging.getLogger(__name__).error(
```

Comment:
Maybe it would be better to define the logger at the top of the module, so it doesn't need to be created both here and on line 58?